tests_e2e/GuestAgentDcrTestExtension/.gitignore

```diff
@@ -0,0 +1,71 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
do we need this file on top of the main .gitignore?

Reply:
no, I'll remove this in next commit